pyproject.toml

```
@@ -31,12 +32,14 @@ classifiers = [
   "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-  "anyio>=3",
+  "anyio>=3 ; python_version < \"3.9\"",
+  "anyio>=4 ; python_version >= \"3.9\"",
```

I think this needs to be 

```suggestion
  "anyio>=3 ; python_version < \"3.13\"",
  "anyio>=4 ; python_version >= \"3.13\"",
```

---

It's been reverted to just `anyio>=3`